### PR TITLE
Remove places that mentions Python 3.5 support due to EOL

### PR DESCRIPTION
**Pull request by Superskyyy**
Will test Python 3.10 when officially releases next month.

Signed-off-by: Superskyyy <Superskyyy@outlook.com>

### CODE REVIEW

**Review by kezhenxu94 — APPROVED** [review]

**Comment by kezhenxu94** [comment]
We can say 3.5 is EOL and is not in our test matrix, it works in versions prior to 1.0, but it may or may not work with the codebase evolves.

**Comment by Superskyyy** [comment]
> We can say 3.5 is EOL and is not in our test matrix, it works in versions prior to 1.0, but it may or may not work with the codebase evolves.

Ok, I intend to swap existing string formatting(`%` `.format` `+` etc.) to f-strings since they are much faster, but that does blocks all Python 3.5 users from v1.0.0, should I do that?

**Comment by kezhenxu94** [comment]
> > We can say 3.5 is EOL and is not in our test matrix, it works in versions prior to 1.0, but it may or may not work with the codebase evolves.
> 
> 
> 
> Ok, I intend to swap existing string formatting(`%` `.format` `+` etc.) to f-strings since they are much faster, but that does blocks all Python 3.5 users from v1.0.0, should I do that?

That's awesome. We can do that, and just say since 1.0 we don't support 3.5.

**Comment by wu-sheng** [comment]
As you said we will drop 3.5 supports, https://twitter.com/ASFSkyWalking/status/1439553219210792963.

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ microservices, cloud native and container-based (Docker, Kubernetes, Mesos) arch
 
 ## Installation Requirements
 
-SkyWalking Python Agent requires SkyWalking 8.0+ and Python 3.5+.
+SkyWalking Python Agent requires SkyWalking 8.0+ and Python 3.6+.
```
Comment (kezhenxu94) [inline README.md:22]: OK, since the requirement here is version-ed, users should check the corresponding doc of their versions, this PR is good to me now

**Review by kezhenxu94 — APPROVED** [review]

**Comment by kezhenxu94** [comment]
> Will test Python 3.10 when officially releases next month.
> 

Seems 3.10 is out. @Superskyyy

**Pull request merged**